src/Integration/Binding/CSharpVBBindingConfigProvider.cs

```diff
@@ -109,13 +109,23 @@ private async Task<IBindingConfig> DoGetConfigurationAsync(SonarQubeQualityProfi
 
             // Finally, fetch the remaining data needed to build the ruleset
             var inactiveRules = await FetchSupportedRulesAsync(false, qualityProfile.Key, cancellationToken);
+            var exclusions = await FetchInclusionsExclusionsAsync(bindingConfiguration.Project.ProjectKey, cancellationToken);
 
             var ruleset = GetRulesetFile(qualityProfile, language, bindingConfiguration, activeRules, inactiveRules, sonarProperties);
-            var additionalFile = GetAdditionalFile(language, bindingConfiguration, activeRules, sonarProperties);
+            var additionalFile = GetAdditionalFile(language, bindingConfiguration, activeRules, sonarProperties, exclusions);
 
             return new CSharpVBBindingConfig(ruleset, additionalFile);
         }
 
+        private async Task<ServerExclusions> FetchInclusionsExclusionsAsync(string projectKey,
```

Comment:
We do this exact call in BindingProcessImpl, to generate sonar.settings.json.
I think since they're different format and files, it's simpler to just re-fetch it.

Reply:
We've actually already fetch all of the properties for the project in line 101:
```
var sonarProperties = await FetchPropertiesAsync(bindingConfiguration.Project.ProjectKey, cancellationToken);
```

... which include the `sonar.exclusions` etc. It's something we could perhaps simplify in the future.